packages/bpk-component-barchart/src/__snapshots__/BpkBarchartBar-test.js.snap

```diff
@@ -23,8 +23,9 @@ exports[`BpkBarchartBar should render as an outlier 1`] = `
     className="bpk-barchart-bar__tappable-area"
     height={100}
     onClick={undefined}
-    onHover={undefined}
+    onFocus={undefined}
```

Comment:
The `onFocus` is a little strange - what was the linter warning you mentioned that made you add this?

Reply:
It's a usability linter error: 
> [eslint] onMouseOver must be accompanied by onFocus for accessibility. ([jsx-a11y/mouse-events-have-key-events](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/mouse-events-have-key-events.md))

Reply:
Ah right - gotcha. In this case I think it's wrong to attach the `onBarHover` to the `onFocus` event because from a principle of least surprise POV it's .... surprising :P

I.e. imagine you wanted to show a tooltip on hover... this would inadvertently show it when you focused on the bar too. This should be an entirely separate event handler - that may be the best fix here.

Reply:
Good point 🙂  added this new event